types/promise.any/OTHER_FILES.txt

```diff
@@ -0,0 +1,2 @@
+auto.d.ts
```

Comment:
```txt
auto.d.ts
```
same here, it does nothing, doesn't it? Why do we need this file

Reply:
If I understand correctly, users can import auto.js to avoid having to import shim.js and then call the resulting function. auto.js just polyfills `Promise.any` as an import side-effect. I included auto.d.ts in OTHER_FILES.txt because none of the other typings files import it.